src/map/status.cpp

```diff
@@ -9299,7 +9299,7 @@ struct status_change *status_get_sc(struct block_list *bl)
  */
 void status_change_init(struct block_list *bl)
 {
-	struct status_change *sc = status_get_sc(bl);
+	status_change *sc = status_get_sc(bl);
 	nullpo_retv(sc);
 	memset(sc, 0, sizeof (struct status_change));
```

Comment:
```suggestion
	memset(sc, 0, sizeof (status_change));
```

Reply:
I'm not the person who refactored the statuses, do you want me to do those fixes here?

Reply:
Yeah, it was just a couple other warnings that are popping up from the change so might as well fix it now!